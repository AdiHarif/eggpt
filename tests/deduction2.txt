Cats are afraid of wolves.
Mice are afraid of cats.
Sheep are afraid of mice.
Gertrude is a cat.
Wolves are afraid of sheep.
Jessica is a mouse.
Emily is a wolf.
Winona is a cat.
What is emily afraid of?
What is emily afraid of?
What is jessica afraid of?
What is gertrude afraid of?