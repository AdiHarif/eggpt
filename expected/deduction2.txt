What is emily afraid of?	sheep	7 5
What is emily afraid of?	sheep	7 5
What is jessica afraid of?	cat	6 2
What is gertrude afraid of?	wolf	4 1